src/plugins/intel_cpu/src/graph_dumper.cpp

```diff
@@ -164,8 +164,11 @@ std::shared_ptr<ov::Model> dump_graph_as_ie_ngraph_net(const Graph &graph) {
             results.emplace_back(std::make_shared<ov::op::v0::Result>(get_inputs(node).back()));
             return_node = results.back();
         } else {
+            // Even though outConfs.size() equals childEdges.size() during initailization,
+            // CPUNode's children node may be changed by graphOptimizer such as ShareReOrder
+            // The output_size here must use actual size of childEdges, otherwise out-of-index happens in getChildEdgeAt
```

Comment:
If I read this comment correctly ShareReorder transformation breaks port configuration. I would say this is not expected behavior and should be fized inside the transformation itself.
@maxnick  @usstq guys, could you please comment?

Reply:
The problem here is that `outConfs` is configured inside  `InitOptimalPrimitiveDescriptors`, when `ShareReorders`([link](https://github.com/openvinotoolkit/openvino/blob/5c0d1468d53a1908a566b8a1bac41ae5dba72462/src/plugins/intel_cpu/src/graph.cpp#L248)) removes the edges from broadcast to duplicated Reorders, it only shrinks childEdges, there is no API to also configure the `outConfs`

Reply:
Than such API should be added. 
Port descriptors and edges configuration should be consistent during whole model compilation/execution